src/SendGrid/SendGridClient.cs

```diff
@@ -126,45 +126,45 @@ public enum Method
             /// <summary>
             /// Remove a resource.
             /// </summary>
-            DELETE,
+            Delete,
```

Comment:
These are breaking changes. If we refactor this, we should consider just using the `HttpMethod` type in the BCL instead of defining out own.

Reply:
So shall we change this to use `HttpMethod` ? @thinkingserious ??